src/ui/components/KeyboardShortcuts.tsx

```diff
@@ -152,7 +154,8 @@ function KeyboardShortcuts({
       // Can pre-fill the dialog with specific filter prefixes
       "CmdOrCtrl+Shift+O": toggleFunctionQuickOpenModal,
       "CmdOrCtrl+O": toggleProjectFunctionQuickOpenModal,
-
+      "CmdOrCtrl+Z": jumpToLastPause,
+      "Shift+CmdOrCtrl+Z": jumpToNextPause,
```

Comment:
cmd+z / cmd+shift+z is an established shortcut key combo. I'm not sure if it makes sense to use it for stepping like this. Not sure what prior art we could use for this though, since it's kind of a Replay-specific concept.

cc @jonbell-lot23

Reply:
Thank you @bvaughn for the review! I agree `cmd+z / cmd+shift+z` is an established shortcut key combo. If we have to find the alternative we can go with this idea. Generally for video play back we use `j` (rewind), `k`(pause), `l` (fast-forwarding). So similarly we can use `shift/cmd + j` ( to step back to the previous pause ) and `shift/cmd + l` ( to step front to the next pause ). Since this feature is Replay-specific concept I feel we can introduce this as new feature for future debugging.

Reply:
on interesting. I guess we could borrow from linear

<img width="406" alt="Screenshot 2023-03-06 at 10 44 38 AM" src="https://user-images.githubusercontent.com/254562/223202190-9def88f4-286a-41d1-8e6c-e2b1c1475541.png">

Reply:
Haven't worked with Linear much. But have seen, it's a pretty common productivity tool. I like the idea of borrowing it from linear.

Reply:
Youtube uses `option + ⬅` to go back to the previous chapter. We can consider chapters as pauses and use it?
<img width="605" alt="Screenshot 2023-03-07 at 12 44 43 AM" src="https://user-images.githubusercontent.com/83641534/223208209-0b9be9c4-e57e-44a8-a457-4b8264aa747e.png">

Reply:
Hey team! Please [read the background here](https://github.com/replayio/devtools/discussions/3885#discussioncomment-5221973).

In that discussion, I recommend against command-Z and option-arrow and instead propose command-[ because it's the universal standard for moving backwards through history. (Which is why Linear does it)

Note: I tried the preview branch with the command-Z functionality and it doesn't work reliably for me. When we do command-[ we should make sure it is jumping forward and backward correctly.

Reply:
Hey guys! So as per this [discussion](https://github.com/replayio/devtools/discussions/3885#discussioncomment-5233276), I was doing the required changes. I found out that `Cmd+[` and `Cmd+]` are already reserved for traversing back and next page in chrome browser. So I feel we still need to figure out some other alternative.
<img width="1440" alt="Screenshot 2023-03-08 at 1 10 15 PM" src="https://user-images.githubusercontent.com/83641534/223651854-d7429f23-f7c4-4203-bd91-36b7657f97b7.png">

Reply:
Okay cool I will do that 🙌